templates/default/activities.html.twig

```diff
@@ -11,7 +11,10 @@
 
 {% block infoline %}
     {% set infoline = {
-        'label': _Tn("%1$s activity", "%1$s activities", nb, "events")|replace({"%1$s": nb}),
+        'label': _Tn("%count activity", "%count activities", nb, "events")|replace({"%count": nb}),
```

Comment:
I've recently changed all those translations; because several translators did not understand this is a replacement string an try to translate it.

Reply:
I've reverted label on changes